grails-app/controllers/org/bbop/apollo/AnnotationEditorController.groovy

```diff
@@ -903,8 +903,8 @@ class AnnotationEditorController extends AbstractApolloController implements Ann
             return
         }
         JSONObject featureContainer = createJSONFeatureContainer()
-        JSONObject sequenceObject = sequenceService.getSequenceForFeatures(inputObject)
-        featureContainer.getJSONArray(FeatureStringEnum.FEATURES.value).put(sequenceObject)
+        JSONArray featuresArray = requestHandlingService.getSequenceForFeatures(inputObject)
```

Comment:
Mostly because RHS is overloaded already and most of the other sequence request stuff goes through SequenceService